RELEASE-NOTES.md

```diff
@@ -1,5 +1,81 @@
 # Prefect Release Notes
 
+## Release 2.7.2
+
+### Extensions to the new pausing feature!
+
+Pausing flow runs with Prefect has become much more powerful!
+
+Flow runs can now be paused without blocking the flow run process by passing `reschedule=True` to the `pause_flow_run` function. This feature can free up agents to run other flows while waiting for approval. In order to use rescheduling pauses, flows must be deployed and results must be configured with the `persist_result` option.
```

Comment:
nit: This doesn't free up agents. This frees up infrastructure though. Maybe focus on cost reduction?